app/decorators/action_links/link_builder.rb

```diff
@@ -56,11 +57,11 @@ def method_for_action(action)
       end
     end
 
-    def url_for_action(action, method = :get)
+    def url_for_action(action, method = :get, format = :html)
```

Comment:
I ended up removing this because I changed the route to a get request which worked. But these link builders feel a lil over built imo, and get confusing! Spent too much time trying to figure it out.